html/changelogs/transmission_method-could-probably-be-enumerated-through-define.would-be-neater.yml

```diff
@@ -0,0 +1,4 @@
+author: Amunak
+delete-after: True
+changes:
+  - refactor: "Refactors transmission_method values through #define. Is neater."
```

Comment:
don't think players would care about this, i think this PR can go without a changelog

Reply:
Oh can you make a PR without a changelog? Wasn't entirely clear to me.

Reply:
Yeah, if it is something small like this that will have zero impact on players.

Reply:
Alright, removed.